src/service/cid-impl.js

```diff
@@ -170,12 +173,17 @@ export class Cid {
    * @return {!Promise<?string>}
    */
   getExternalCid_(getCidStruct, persistenceConsent) {
+    const scope = getCidStruct.scope;
     /** @const {!Location} */
     const url = parseUrl(this.ampdoc.win.location.href);
     if (!isProxyOrigin(url)) {
+      const apiClient =
+          ViewerCidApi.scopeOptedInForCidApi(this.ampdoc.win, scope);
+      if (apiClient) {
+        return this.cidApi_.getScopedCid(scope, apiClient);
```

Comment:
I am probably not understanding this. Is the cookie name being used to store the client id or are we storing the client id in cookie named after scope?